app/assets/javascripts/components/client/ClientProfile.jsx

```diff
@@ -4,27 +4,140 @@ class ClientProfile extends React.Component {
   constructor(props) {
     super();
     this.state = {
-      users: [],
+      client: {},
     }
   }
 
   componentDidMount() {
-
-    Requester.get('/api/users/').then((data) => {
-      this.setState({users: data});
+    Requester.get('/api/clients/' + this.props.client.id).then((data) => {
```

Comment:
bump on dat routes file doe

Reply:
pls have mercy on us

Reply:
@amyshan @lbkchen do you think it would be best to put the routes file in a separate PR?